cps/src/main/js/samples.js

```diff
@@ -0,0 +1,35 @@
+var samples = {};
+
+exports.getSample = function(sampleName) {
+    var sample = samples[sampleName];
+    if (sample) {
+        return sample;
+    } else {
+        return '';
+    }
+};
+
+samples.hello = "node {\n" +
+    "   stage 'Stage 1'\n" +
+    "   echo 'Hello World 1'\n" +
+    "   stage 'Stage 2'\n" +
+    "   echo 'Hello World 2'\n" +
+    "}";
+
+samples.maven = "node {\n" +
+    "   // Mark the code checkout 'stage'....\n" +
+    "   stage 'Checkout'\n" +
+    "\n" +
+    "   // Checkout some code from a github repository\n" +
```

Comment:
"Checkout" might not be in the dictionary, but isn't it an accepted term for getting code from an SCM repo?